src/d3a/models/strategy/__init__.py

```diff
@@ -258,7 +258,11 @@ def event_response_was_received_callback():
             self.event_response_uuids.remove(data["transaction_uuid"])
 
     def area_reconfigure_event(self, *args, **kwargs):
-        pass
```

Comment:
dont you need the pass anyway?

Reply:
Apparently, you don't need it if you add a docstring. If found that out because pylint complains if you have both a docstring and a `pass` statement in an "empty" method. The difference is that a docstring populates the `__doc__` special attribute of the method (useful to generate documentation) so it's not totally equivalent to the `pass` statement.

Reply:
Interesting!